packages/mdc-select/index.js

```diff
@@ -264,6 +264,7 @@ class MDCSelect extends MDCComponent {
       // will return false if the menu is still closing and this method listens to the closed event which
       // occurs after the menu is already closed.
       this.menuOpened_ = false;
+      this.selectedText_.removeAttribute('aria-expanded');
```

Comment:
I think we want to be setting this to false, not removing it.

> **true/false/undefined**
> Value representing true, false, or not applicable. For example, an element with aria-expanded set to false is not currently expanded; an element with aria-expanded set to undefined is not expandable. The default value for this value type is undefined unless otherwise specified.

Reply:
The example I linked above indicates that `aria-expanded` is not present when the popup is closed. Also, from the `menu` spec, which is closely associated with the listbox example we're implementing: 

> When the menu is displayed, the element with role button has aria-expanded set to true. When the menu is hidden, it is recommended that aria-expanded is not present. If aria-expanded is specified when the menu is hidden, it is set to false.

From what I gather `aria-expanded` should be used to indicate a control is in an expanded (or displayed) state but when focus has not moved from the original element. When focus moves into the popup, the expanded attribute should not be used (because it's ignored by screen readers since focus has moved). It is only used when the menu is open to accommodate for touch devices where screen reader users can touch the menu button when the menu is displayed. Keyboard users cannot focus the menu when the menu is open. 

https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html
https://www.w3.org/TR/wai-aria-practices/#menubutton